configurations/importer.py

```diff
@@ -127,9 +127,9 @@ def stylize(text):
             if (self.argv[1] == 'runserver'
                     and os.environ.get('RUN_MAIN') == 'true'):
 
-                message = ("django-configurations version {0}, using "
-                           "configuration {1}".format(__version__ or "",
-                                                      self.name))
+                message = ("django-configurations version {}, using "
```

Comment:
IMO (personal, admittedly) the changes in this file are not worth changing blame.  Changes of lines that are 90% syntax juice that can now be simplified makes perfect sense, but user-visible and developer-visible strings tend to be something that people do use to navigate back through the history, especially if the string actually modifies later in the project history, and breaks in blame are a tiny annoyance.

The codebase is already using `{}`.  Getting rid of the remaining `{n}` isnt related to dropping support for Python 2, so bloats this PR.

Probably just a byproduct of running an aggressive rewrite tool.  I can guess that because even this diff shows `%s`/`%r` style string formatting with tuples of args.  They arn't being updated.  Why only update one style of string formatting?  All are legal in Python 3.  Because code rewrite tools typically struggle parsing and rewriting the much more horrid syntax of `%s`/`%r` style (been there; done that), and so they dont have that feature.

Reply:
agree with your insight